packages/effects-common/src/mod.ts

```diff
@@ -22,7 +22,7 @@ export class UnhandledEffectError extends TypeError {
   constructor({ type }: Effect) {
     super();
 
-    this.message = `Encountered an unhandled effect :${type}`;
+    this.message = `Encountered an unhandled effect :${String(type)}`;
```

Comment:
huh.

```
[0,null, undefined, 'str', NaN, 1e20, true, false, Infinity, Symbol("wut")].map(thing => ([String(thing)]))
```
works, but not in string templates.  _weirt_.

Reply:
Yeah. Super annoying.